extensions/ahc-ws/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,32 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+name: "Camel Quarkus Async HTTP Client (AHC) Web Socket"
+description: "Camel Quarkus Async HTTP Client (AHC) Web Socket support"
+metadata:
+  keywords:
+  - "camel"
+  - "http"
+  - "websocket"
+  - "websockets"
+  - "web-socket"
+  - "web-sockets"
```

Comment:
Is there something that would break without having all these spelling variants here? (Should we start doing it everywhere?)

Reply:
It's a lazy copy / paste from the Quarkus Undertow Websocket extension. If we really want to trim down this list, I can do follow up PR.

Reply:
Not a big issue, but I'd vote for reducing the list unless we know the expanded list solves some problem.